integration/build/build_userns_linux_test.go

```diff
@@ -66,17 +68,10 @@ func TestBuildUserNamespaceValidateCapabilitiesAreV2(t *testing.T) {
 		})
 	assert.NilError(t, err)
 	defer resp.Body.Close()
-	buf := make([]byte, 1024)
-	for {
-		n, err := resp.Body.Read(buf)
-		if err != nil && err != io.EOF {
-			t.Fatalf("Error reading ImageBuild response: %v", err)
-			break
-		}
-		if n == 0 {
-			break
-		}
-	}
+
+	buf := bytes.NewBuffer(nil)
+	err = jsonmessage.DisplayJSONMessagesStream(resp.Body, buf, 0, false, nil)
+	assert.NilError(t, err)
```

Comment:
After discussing with @tonistiigi @tiborvass - updated this to use jsonmessage to detect errors (instead of checking for "Successfully built"

Possibly there's other tests that have this same issue (so something to look into  in a follow-up)